src/core/thread/network_data.cpp

```diff
@@ -1010,8 +1009,7 @@ otError NetworkData::SendServerDataNotification(uint16_t aRloc16)
 
     if (mType == kTypeLocal)
     {
-        mLastAttempt     = TimerMilli::GetNow();
-        mLastAttemptWait = true;
+        mLastAttempt = TimerMilli::GetNow();
```

Comment:
What is `TimerMilli::GetNow()` returns `0`?

Reply:
In case there is wrap and here returns 0 after ~49days, the impact is that there's chance that be two successive SRV_DATA.ntf in a period shorter than the expected delay.
I am fine to recover if you prefer previous way.

Reply:
I'm OK with one of the following:

1. Check for 0 and set it to 1 if so.

2. Add a comment that shorter delay may occur if `TimerMilli::GetNow()` returns `0`.

Reply:
Thanks for the suggestion, I like option 1. will update.